src/gaussjacobi.jl

```diff
@@ -426,3 +600,99 @@ function jacobi_gw(n::Integer, α, β)
     w = V[1,:].^2 .* jacobimoment(α, β)
     return x, w
 end
+
+function bary(x::AbstractVector, fvals::AbstractVector, xk::AbstractVector, vk::AbstractVector)
+    # simple barycentric interpolation routine adapted from chebfun/bary.m
+
+    # Initialise return value:
+    fx = zeros(length(x))
+
+    # Loop:
+    for j in eachindex(x)
+        xx = vk ./ (x[j] .- xk)
+        fx[j] = dot(xx, fvals) / sum(xx)
+    end
+
+    # Try to clean up NaNs:
+    for k in findall(isnan.(fx))
+        index = findfirst(x[k] .== xk) # Find the corresponding node
+        if !isempty(index)
+            fx[k] = fvals[index]       # Set entries to the correct value
+        end
+    end
+
+    return fx
+end
+
+function besselTaylor(t::AbstractVector, z::AbstractVector, α::Float64)
```

Comment:
We don't use `lowerCamelCase` for function names, so `besseltaylor` or `bessel_taylor` would be preferred.
https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/

Reply:
`besselTaylor` was just used to agree with the Chebfun MATLAB source - I'm happy to change it to `besseltaylor`.

Reply:
note that `HalfRec` and `weightsConstant`, which were already in `gaussjacobi.jl`, are also camel case.

Reply:
Ah, that's right. `HalfRec` and `weightsConstant` existed before I started contributing. I prefer renaming these functions too (in some other PR).